SWLOR.Game.Server/Feature/AbilityDefinition/ThrowLightsaberAbilityDefinition.cs

```diff
@@ -0,0 +1,178 @@
+﻿//using Random = SWLOR.Game.Server.Service.Random;
+using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.Creature;
+using SWLOR.Game.Server.Core.Bioware;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.AbilityService;
+using static SWLOR.Game.Server.Core.NWScript.NWScript;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+
+namespace SWLOR.Game.Server.Feature.AbilityDefinition
+{
+    public class ThrowLightsaberAbilityDefinition : IAbilityListDefinition
+    {
+        public Dictionary<Feat, AbilityDetail> BuildAbilities()
+        {
+            var builder = new AbilityBuilder();
+            ThrowLightsaber1(builder);
+            ThrowLightsaber2(builder);
+            ThrowLightsaber3(builder);
+            ThrowLightsaber4(builder);
+            ThrowLightsaber5(builder);
+
+            return builder.Build();
+        }
+
+        private static void SaberThrow(uint activator, uint target, int level)
+        {
+            uint weapon = GetItemInSlot(InventorySlot.RightHand, activator);
```

Comment:
Just a style nit: try to stick to var. Makes refactoring a little easier if we don't have to deal with data types changing.

Reply:
done